.gitignore

```diff
@@ -1,3 +1,6 @@
+# Setuptools SCM version.py file
+pydecorate/version.py
```

Comment:
Wasn't the file deleted in this PR ?

Reply:
It is auto-generated by setuptools_scm when you install the package.

Reply:
ok, I understand then.